src/main.c

```diff
@@ -668,18 +796,18 @@ char aes_decrypt(unsigned char *input,
     // set the number of rounds
     switch (size)
     {
-    case SIZE_16:
-        nbrRounds = 10;
-        break;
-    case SIZE_24:
-        nbrRounds = 12;
-        break;
-    case SIZE_32:
-        nbrRounds = 14;
-        break;
-    default:
-        return ERROR_AES_UNKNOWN_KEYSIZE;
-        break;
+        case SIZE_16:
+            nbrRounds = 10;
+            break;
+        case SIZE_24:
+            nbrRounds = 12;
+            break;
+        case SIZE_32:
+            nbrRounds = 14;
+            break;
+        default:
+            return ERROR_AES_UNKNOWN_KEYSIZE;
+            break;

```

Comment:
Avoid changing the indentation in code that hasn't been modified, as it makes it harder to track the main changes in the review.